assets/section-password.css

```diff
@@ -324,3 +324,29 @@ details.modal .modal__toggle-open {
   vertical-align: top;
   color: rgb(var(--color-foreground));
 }
+
```

Comment:
When some images are not super big and with the text box over them, it can leave some white gap between the section and footer. 

Should we keep `flex-grow` on the `main` element ? 🤔 

![](https://screenshot.click/21-10-qj863-iaqpr.png)

Reply:
@Oliviammarcello @ludoboludo The reason this was added was to ensure the footer always stays at the bottom (I will try to find the PR and the discussion/reference). Otherwise this would happen: 

This is just this one scenario of a shorter image and only one section.

Video: https://screenshot.click/05-42-3bglx-zh3lg.mp4

Options:

1) remove and have the footer go up (it may look odd because the page on mobile looks short)
2) keep it and ensure that the footer stays at the bottom

Reply:
I think it actually makes sense to have it below because youd want the footer not to be floating and have a lot of white space below all the content 🤔 it makes the page feel short I think which is a weird experience on mobile (also this is an edge case for a short image). Once the heights get adjusted too (Ken's PR), well have a different experience so itll be good to have those changes before

https://screenshot.click/05-01-u33m1-5mu8l.png

Reply:
See on wider screens even images that are fairly big might end up showing something like this which is kinda "meh" I find: [screenshot](https://screenshot.click/21-10-67s2u-drqbb.png)

Reply:
Yea, this definitely looks better if it's a shorter image ... 🤔 Once I rebase we can retest these scenarios